VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/quick-links/quick-links.component.scss

```diff
@@ -0,0 +1,6 @@
+@media (min-width: 40.0625em) {
+  .full-name-text-input {
+    width: 50%;
+    min-width: 25em;
```

Comment:
If we do stick with this solution please change this to rem.

Reply:
And just to be clear on devices from 641-800ish pixels this will be more than half width. Not accounting for padding or margins of parent elements. That's fine but just want it to be clear that Neil is ok with that.

Reply:
when using rem, it appears to have the same effect as using that class that Neil didnt like as well, so shall I stick with rem or keep using the class which makes it look weird in ipad view?

Reply:
em isn't consistant enough for this sort of styling as it could be changed by a change to any of it's ancestor elements. We can change the number for rem if that is a concern.

Also perhaps this should be on the container div rather than the input itself and allow the input to fill the width of it's parent. Perhaps the govuk class being applied there would solve the issues you saw.